### PR TITLE
Doc/update post processing examples

### PR DESCRIPTION
**Pull request by jose-henriquezroa**
Updated distributed post processing examples.

### CODE REVIEW

File: examples/06-distributed-post/00-distributed_total_disp.py [hunk 1]
```diff
@@ -1,94 +1,115 @@
-"""
-.. _ref_distributed_total_disp:
```
Comment (PProfizi) [inline examples/06-distributed-post/00-distributed_total_disp.py:2]: You lost the reference here but not in the other examples, is that on purpose?

File: examples/06-distributed-post/04-distributed_total_disp_with_operator.py [hunk 2]
```diff
@@ -7,6 +7,45 @@
 on distributed processes. After remote post processing, results a merged
```
Comment (PProfizi) [inline examples/06-distributed-post/04-distributed_total_disp_with_operator.py:7]: "are" merged

File: examples/06-distributed-post/00-distributed_total_disp.py (hunk 1 above)
Comment (jose-henriquezroa) [reply examples/06-distributed-post/00-distributed_total_disp.py:2]: No it wasn't intentional. I will add it back.

File: docs/source/conf.py [hunk 3]
```diff
@@ -1,223 +1,227 @@
...
+
+# A unique identification for the text.
+#
+# epub_uid = ''
+
+# A list of files that should not be packed into the epub file.
+epub_exclude_files = ["search.html"]
+
+# GraphViz configuration
+graphviz_output_format = 'svg'
```
Comment (PProfizi) [inline docs/source/conf.py:227]: At the same time as correcting the "are" in the other file, could you please add a newline at the end of this file? It's just to comply with standards, and if this was the only thing to change I would not ask you a new commit just for that.

Comment (jose-henriquezroa) [reply docs/source/conf.py:227]: Understood, I will do that.

**Comment by PProfizi** [comment]
It does not appear to create the graphviz images correctly:
image

**Comment by PProfizi** [comment]
image

Do you think it would be possible to manage the font size in the graphs?
Some images are small and the font is big, some images are bigger and the font gets a lot smaller. It also does not seem to use the whole width of the text?

**Review by PProfizi — APPROVED** [review]

File: docs/source/examples/06-distributed-post/00-distributed_total_disp.py [hunk 4]
```diff
@@ -3,10 +3,13 @@
 
 Distributed post without client connection to remote processes
```
Comment (cbellot000) [inline docs/source/examples/06-distributed-post/00-distributed_total_disp.py:4]: @jose-henriquez-roa could you please rename this example to: "Post processing of displacement on distributed processes"

File: docs/source/examples/06-distributed-post/00-distributed_total_disp.py [hunk 5]
```diff
@@ -16,27 +19,11 @@
...
 ###############################################################################
 # Configure the servers
 # ~~~~~~~~~~~~~~~~~~~~~~
-# Make a list of ip addresses an port numbers on which dpf servers are
-# started. Workflows instances will be created on each of those servers to
+# Make a list of ip addresses and port numbers on which dpf servers are
+# started. Operator instances will be created on each of those servers to
 # address each a different result file.
 # In this example, we will post process an analysis distributed in 2 files,
 # we will consequently require 2 remote processes
```
Comment (cbellot000) [inline docs/source/examples/06-distributed-post/00-distributed_total_disp.py:29]:
```suggestion
# we will consequently require 2 remote processes.
```

File: examples/06-distributed-post/04-distributed_total_disp_with_operators.py [hunk 6]
```diff
@@ -1,14 +1,15 @@
 """
-.. _ref_distributed_delegate_to_server:
+.. _ref_distributed_total_disp_op:
```
Comment (cbellot000) [inline examples/06-distributed-post/04-distributed_total_disp_with_operators.py:2]: if this example is the same as the first one, we can remove this file

File: examples/06-distributed-post/01-distributed_workflows_on_remote.py [hunk 7]
```diff
@@ -0,0 +1,74 @@
+"""
+.. _ref_distributed_workflows_on_remote:
+
+Connect operator chains on different processes implicitly
```
Comment (cbellot000) [inline examples/06-distributed-post/01-distributed_workflows_on_remote.py:4]: Could you please rename this one to: "Create custom workflow on distributed processes."

File: examples/06-distributed-post/02-distributed-msup_expansion.py [hunk 8]
```diff
@@ -7,38 +7,27 @@
 on distributed processes. The modal basis (2 distributed files) is read
 on 2 remote servers and the modal response reading and the expansion is
 done on a third server.
+
+To help understand this example the following diagram is provided. It shows
+the operator chain used to compute the final result.
+
+.. image:: 02-operator-dep.svg
```
Comment (cbellot000) [inline examples/06-distributed-post/02-distributed-msup_expansion.py:14]: in this drawing, could you please add a datasource as an input of "response"

Comment (cbellot000) [reply examples/06-distributed-post/02-distributed-msup_expansion.py:14]: and maybe rename merge as "merge_fields"

File: examples/06-distributed-post/02-distributed-msup_expansion.py [hunk 9]
```diff
@@ -7,38 +7,27 @@
...
 ###############################################################################
 # Configure the servers
 # ~~~~~~~~~~~~~~~~~~~~~~
-# Make a list of ip addresses an port numbers on which dpf servers are
-# started. Workflows instances will be created on each of those servers to
+# Make a list of ip addresses and port numbers on which dpf servers are
+# started. Operator instances will be created on each of those servers to
 # address each a different result file.
 # In this example, we will post process an analysis distributed in 2 files,
 # we will consequently require 2 remote processes
```
Comment (cbellot000) [inline examples/06-distributed-post/02-distributed-msup_expansion.py:33]:
```suggestion
# we will consequently require 2 remote processes.
```

File: examples/06-distributed-post/02-distributed-msup_expansion.py [hunk 10]
```diff
@@ -50,36 +39,41 @@
...
+    displacement.inputs.data_sources(ds)
+    mesh.inputs.data_sources(ds)
 
 ###############################################################################
-# Create a local workflow for expansion
+# Create a local operators chain for expansion
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# In this workflow we merge the modal basis, the meshes, read the modal response
-# and expand the modal response with the modal basis
+# In the follwing series of operators we merge the modal basis, the meshes, read
```
Comment (cbellot000) [inline examples/06-distributed-post/02-distributed-msup_expansion.py:74]:
```suggestion
# In the following series of operators we merge the modal basis, the meshes, read
```

File: .ci/build_doc.bat [hunk 11]
```diff
@@ -2,6 +2,6 @@ set SPHINX_APIDOC_OPTIONS=inherited-members
 call sphinx-apidoc -o ../docs/source/api ../ansys -f --implicit-namespaces --separate  --no-headings
 pushd .
 cd ../docs/
-call make clean
+:: call make clean
```
[the remark this answers is not in the document]
Comment (jose-henriquezroa) [reply .ci/build_doc.bat]: I did this back when I had yet to ask whether there was a way to get around the documentation build timeout error. I was just testing whether removing the make clean would reduce the build time, I added it back afterwards.

**Review by cbellot000 — APPROVED** [review]

**Pull request merged**